src/frontend/typer/behandlingstema.ts

```diff
@@ -0,0 +1,39 @@
+export type Behandlingstema =
```

Comment:
Dette er egentlige saker som jag syns backend borde kunne mappe om slik att frontend får ett finere grensesnitt og koder..

Reply:
Enig!

Reply:
Ja, men da må vi isåfall endre i kontrakter+integrasjoner slik at oppgavene vi mottar inneholder "vår" verdi. Husk at denne listen egentlig ikke er uttømmende. Det finnes mange flere enn disse i oppgavesystemet...